src/Helper/WebformHelper.php

```diff
@@ -50,4 +39,24 @@ public function webformThirdPartySettingsFormAlter(array &$form, FormStateInterf
     ];
   }
 
-}
+  /**
+   * Define the email templates to use for sending emails through this module.
+   *
+   * @param $key
+   *   The mail key.
+   * @param $message
+   *   The message to send.
+   * @param $params
+   *   Various parameters used by the mail template.
+   *
+   * @return void
+   */
+  public function setEmailTemplate($key, &$message, $params) {
```

Comment:
Consider renaming this function to `mail` since it implements `hook_mail()`.

Reply:
Changed